ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vxlan-interface.j2

```diff
@@ -1,20 +1,35 @@
-{# eos- VxLAN interface #}
+{# eos- VxLAN Interface #}
 {% if vxlan_tunnel_interface is arista.avd.defined %}
```

Comment:
since the interface is statically set below, this `if` should be testing for `vxlan_tunnel_interface.Vxlan1`

Reply:
addressed with commit 5e43333